engine/apps/grafana_plugin/views/sync.py

```diff
@@ -25,12 +27,13 @@ def post(self, request: Request) -> Response:
 
         try:
             organization = Organization.objects.get(stack_id=stack_id, org_id=org_id)
-
-            if organization.api_token_status == Organization.API_TOKEN_STATUS_OK:
-                is_installed = True
-            organization.api_token_status = Organization.API_TOKEN_STATUS_PENDING
-
-            organization.save(update_fields=["api_token_status"])
+            try:
```

Comment:
Could you please add comments here? I didn't catch the meaning of that check and why we setting API_TOKEN_STATUS_PENDING if we failed to get user from grafana context.

Reply:
For all the other cases (user does not exist, organisation token "not ok", etc) process remains same - plugin will show "Initialising plugin..." until the background task in successfully completed

Reply:
Yep, but what is the case you are checking here and what is the expected behaviour? I mean, in which case we are unable to get user from grafana context?

Reply:
btw, cleaned up a little and added pr description
you were too fast:)

Reply:
fixed is_installed part, thanks